deployment/updates/scripts/xml/notifications/entryClipJobCompletedHttpNotificationTemplate.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xml>
+    <config>
+        <serviceUrl>@SERVICE_URL@</serviceUrl>
+        <partnerId>-2</partnerId>
+        <clientTag>Stand alone php 1.0.0</clientTag>
+        <curlTimeout>30</curlTimeout>
+        <userAgent>Stand alone php 1.0.0</userAgent>
+    </config>
+    <session>
+        <partnerId>-2</partnerId>
+        <secret>@ADMIN_CONSOLE_PARTNER_ADMIN_SECRET@</secret>
+        <sessionType>2</sessionType>
+    </session>
+    <multirequest>
+        <request service="eventNotificationTemplate" action="add" plugin="eventNotification" partnerId="0">
+            <template objectType="KalturaHttpNotificationTemplate">
+                <name>Entry Clip Job Finished</name>
+                <systemName>HTTP_ENTRY_CLIP_JOB_FINISHED</systemName>
```

Comment:
@TalDubovKaltura  I think a better name and systemName would be 
Entry Trimming Process Finished
and
HTTP_ENTRY_TRIM_JOB_FINISHED
Since this solution will probably not work for clips.